### PR TITLE
Adapt cli commands for rebar 3.14

### PR DESCRIPTION
**Pull request by Vagabond**
Related to helium/miner#567

### CODE REVIEW

**Comment by Vagabond** [comment]
One problem here is all the command outputs now ends with the line `ok` which breaks some scripts that expect the output on a single line.

**Comment by Vagabond** [comment]
This depends on erlware/relx#855

**Comment by Vagabond** [comment]
Additionally this branch works with OTP 24

File: rebar.config [hunk 1]
```diff
@@ -29,7 +29,7 @@
 }.
 
 {deps, [
-    {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {branch, "master"}}},
+    {lager, "3.9.0"},
```
Comment (mrallen1) [inline rebar.config:32]: is there a reason you don't want to use 3.9.1?

File: src/cli/blockchain_console.erl [hunk 2]
```diff
@@ -4,4 +4,22 @@
 
 -spec command([string()]) -> ok.
 command(Cmd) ->
-    clique:run(Cmd).
+    %% this is the contents of clique:run but
+    %% we want to figure out if the command worked
+    %% or not
```
Comment (mrallen1) [inline src/cli/blockchain_console.erl:9]: This is ok for now but seems like it ought to be a new function in clique `run_and_return_status/1` or something...

**Review by mrallen1 — APPROVED** [review]

File: rebar.config (hunk 1 above)
Comment (Vagabond) [reply rebar.config:32]: this PR is old, I will bump it

File: src/cli/blockchain_console.erl (hunk 2 above)
Comment (Vagabond) [reply src/cli/blockchain_console.erl:9]: yes, we should make an upstream PR

**Review by evanmcc — APPROVED** [review]

**Pull request merged**